tests/CitizenProject/CitizenProjectManagerTest.php

```diff
@@ -121,6 +123,33 @@ public function testChangePrivilege()
         $this->assertEquals(false, $adherent2->getCitizenProjectMembershipFor($citizenProject)->isFollower());
     }
 
+    public function testFindAdherentNearCitizenProjectOrAcceptAllNotification()
+    {
+        $citizenProject = $this->getCitizenProjectRepository()->findOneByUuid(LoadCitizenProjectData::CITIZEN_PROJECT_1_UUID);
+        $adherents = $this->citizenProjectManager->findAdherentNearCitizenProjectOrAcceptAllNotification($citizenProject);
+
+        $this->assertSame(3, $adherents->count());
+
+        $adherents = $this->citizenProjectManager->findAdherentNearCitizenProjectOrAcceptAllNotification($citizenProject, 0, false);
+
+        $this->assertSame(4, $adherents->count());
+
+        $adherent = $this->getAdherentRepository()->findByEmail('francis.brioul@yahoo.com');
+        $adherent->setCitizenProjectCreationEmailSubscriptionRadius(AdherentEmailSubscription::DISTANCE_100KM);
+        $this->getManagerRegistry()->getManager()->persist($adherent);
```

Comment:
This is not needed since this Adherent entity is already managed.

Reply:
✔️